PSReadLine/Render.cs

```diff
@@ -799,6 +799,40 @@ private int ConvertLineAndColumnToOffset(Point point)
             return (point.Y == y) ? offset : -1;
         }
 
+        /// <summary>
+        /// Returns the line number under the cursor.
+        /// </summary>
+        /// <returns></returns>
+        private int GetCurrentLine()
+        {
+            var current = _current;
+            var number = 1;
+
+            for (int i = 0; i < current; i++)
+            {
+                if (_buffer[i] == '\n')
+                    number++;
+            }
+            return number;
+
+        }
+
+        /// <summary>
+        /// Returns the number of lines in a multi-line buffer
+        /// </summary>
+        /// <returns></returns>
+        private int GetLineCount()
```

Comment:
This should probably be named `GetLogicalLineCount` so it isn't used accidentally.

When rendering, a logical line may span multiple actual lines (see [PhysicalLineCount](https://github.com/lzybkr/PSReadLine/blob/bc40ae6f6b44e2eb4bd8b57810079bfd89bffa72/PSReadLine/Render.cs#L391)), so it's useful to know which kind of line count it is.